dotcom-rendering/scripts/islands/descriptions.mjs

```diff
@@ -0,0 +1,146 @@
+// @ts-check
+
+import { readdir, readFile, writeFile } from 'node:fs/promises';
+import { resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import rehypeAutolinkHeadings from 'rehype-autolink-headings';
+import rehypeSlug from 'rehype-slug';
+import rehypeStringify from 'rehype-stringify';
+import remarkParse from 'remark-parse';
+import remarkRehype from 'remark-rehype';
+import { unified } from 'unified';
+import * as zod from 'zod';
+
+/** @type {(name: string) => RegExp} */
+export const get_JSDoc = (name) =>
```

Comment:
strange naming convention?

Reply:
Ah sorry… came from Deno where snake_case is the default and [a study has observed quicker scanning with it](http://www.cs.kent.edu/~jmaletic/papers/ICPC2010-CamelCaseUnderScoreClouds.pdf)

Will revert!